Makefile

```diff
@@ -54,10 +54,18 @@ install_quicklogic:
 	tar -xf ${QUICKLOGIC_ARCHIVE} -C env/quicklogic
 	rm ${QUICKLOGIC_ARCHIVE}
 
-PYTHON_SRCS=$(shell find . -name "*py" -not -path "./third_party/*" -not -path "./env/*" -not -path "./conf/*")
-
-format: ${PYTHON_SRCS}
-	yapf -i $?
+PYTHON_SRCS=$(shell find . -name "*.py" -not -path "./third_party/*" -not -path "./env/*" -not -path "./conf/*")
+JSON_SRCS=$(shell find . -name "*.json" -not -path "./third_party/*" -not -path "./env/*" -not -path "./conf/*")
+VERILOG_SRCS=$(shell find . -name "*.v" -not -path "./third_party/*" -not -path "./env/*" -not -path "./conf/*")
+
+format:
+	yapf -i ${PYTHON_SRCS}
+	for i in ${JSON_SRCS}; do \
+		echo "$$(mjson $$i)" > $$i; \
+	done
+	for i in ${VERILOG_SRCS}; do \
+		echo "$$(verible-verilog-format $$i)" > $$i; \
+	done
```

Comment:
Given that mjson and verible are added to the conda environment, I believe that the environment needs to be activated before formatting (I see in CI that verible and mjson cannot be found)

Reply:
How do I go about activating the environment in the CI test?

Reply:
I think sourcing the env.sh in the ci.yml before calling `make format` should work

Reply:
Ok, that did work. I added make env and then source env.sh.